docs/source/getting_started.rst

```diff
@@ -4,42 +4,72 @@
 Getting Started
 ===============
 
-
----------------
-Prerequisites
----------------
+------------
+Dependencies
+------------
 
 PyNWB has the following minimum requirements, which must be installed before you can get started using PyNWB.
 
-#. Python 3.x
-
-   * Before moving forward, make sure `setuptools <https://pypi.python.org/pypi/setuptools>`_ is installed with your version of Python.
+#. Python 2.7 or Python 3.6
```

Comment:
The good news is that pip will take care of installing the dependencies.

This means we should move the list of dependencies further down the document.